06.Lesson_ThinkingProperties/ThinkingPropertiesExercise/MeetingScheduler/README.md

```diff
@@ -0,0 +1,35 @@
+
+- ❓ Is there a way to check that a Meeting always starts as UNITIALIZED?
+- Unit test that confirms that the start and end times provided when creating a meeting are properly saved onto the meeting
+- Check that only the owner can cancel the meeting
+    - ❓ In my test for this (`onlyOwnerCanCancel`), I do a `scheduleMeeting` so that I have a handle on the meeting owner, and then check that `cancelMeeting` can only be called by the owner. But I'm limiting myself to cases where they are called back-to-back, so this test is not as generic as I would like. Would it be better to "harness" a getter for the owner, so that I can have the meeting start in a more arbitrary state.
+    - ❓ My test for this is failing, can you tell why?
```

Comment:
For failing tests please provide a report link. I can not tell from the rule and the code an obvious reason why this would fail.

Reply:
`require e2.msg.value == 0;` fixed this one